app/components/primer/beta/auto_complete.rb

```diff
@@ -32,14 +32,41 @@ class AutoComplete < Primer::Component
         Primer::BaseComponent.new(**system_arguments)
       }
 
+      # Customizable input used to search for results.
+      # It is preferred to use this slot sparingly - it will be created by default if not explicity added.
+      #
+      # @param system_arguments [Hash] <%= link_to_system_arguments_docs %>
+      renders_one :input, lambda { |**system_arguments|
+        deny_tag_argument(**system_arguments)
+        deny_single_argument(system_arguments[:autofocus], "autofocus is not allowed for accessibility reasons. See https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autofocus#accessibility_considerations for more information.", **system_arguments)
+        deny_single_argument("aria-label", "instead of `aria-label`, include `label_text` and set `is_label_visible` to `false`.", **system_arguments)
```

Comment:
```suggestion
        deny_single_argument("aria-label", "instead of `aria-label`, include `label_text` and set `is_label_visible` to `false` on the component initializer.", **system_arguments)
```